scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -80,8 +80,24 @@ <h3 class="header">Institute Settings</h3>
             </div>
             <!-- End of general settings -->
 
-            <!-- Phenotype groups settings -->
+            <!-- Show cases settings -->
+            {% if "admin" in current_user.roles %}
             <div class="row mt-3 d-flex align-items-center">
+              <fieldset>
+                <legend>Cases display settings</legend>
+                <div class="row">
+                  <div class="col-4">
+                    {{ form.show_all_cases_categories.label(class="control-label") }}
+                    {{ form.show_all_cases_categories }}
```

Comment:
The other dropdown buttons on the page use `class="btn-secondary"`!

![Screenshot 2024-02-09 at 09 35 57](https://github.com/Clinical-Genomics/scout/assets/758570/0d9ce547-ffc4-4372-90f2-8bd40c56cdfc)
![Screenshot 2024-02-09 at 09 35 26](https://github.com/Clinical-Genomics/scout/assets/758570/47b26101-fe66-4e17-9657-496568aba8a4)

Reply:
Fixed